react/features/authentication/components/web/LoginDialog.tsx

```diff
@@ -134,9 +134,7 @@ class LoginDialog extends Component<IProps, IState> {
         if (conference) {
             dispatch(authenticateAndUpgradeRole(jid, password, conference));
         } else {
-            // dispatch(connect(jid, password));
-            // FIXME: Workaround for the web version. To be removed once we get rid of conference.js
-            dispatch(joinConference(undefined, false, jid, password));
+            dispatch(connect(jid, password));
```

Comment:
Has this been tested with XMPP auth?  I recall we've broken it a number of times in the past.